remoteworkitem/flatten.go

```diff
@@ -2,15 +2,24 @@ package remoteworkitem
 
 import (
 	"fmt"
+	"log"
+	"math"
 	"reflect"
 )
 
 // Flatten Takes the nested map and returns a non nested one with dot delimited keys
 func Flatten(source map[string]interface{}) map[string]interface{} {
 	target := make(map[string]interface{})
-
 	flatten(target, source, nil)
-
+	log.Println("Target: ")
+	for k, v := range target {
+		switch v.(type) {
+		case string:
+			value := v.(string)
+			l := int(math.Min(float64(60), float64(len(value))))
+			log.Printf("\t%s=%v\n", k, value[0:l])
```

Comment:
let me remove that statement. It was just about logging the first 60 (max) characters from the string value (some remote work items have huge descriptions and I just wanted the first line, more or less).